<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wiget</class>
 <widget class="QMainWindow" name="wiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wiget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pbCoffee">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>coffee(100)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbTea">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>tea(150)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbCola">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>cola(200)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbReset">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pb500">
            <property name="text">
             <string>500</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb100">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb50">
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb10">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuvending_machine">
    <property name="title">
     <string>vending-machine</string>
    </property>
   </widget>
   <addaction name="menuvending_machine"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
